src/touch/gestures.h

```diff
@@ -39,6 +39,11 @@ typedef struct {
     int32_t velocity;
 } gestures_slider_data_t;
 
+enum button_id_t {
```

Comment:
`bitboxbase_button_id_t`? Or guard it with `#ifdef`s so that we're sure it doesn't affect BB02 stuff.

Reply:
ack